drivers/modem/Kconfig.gsm

```diff
@@ -78,6 +78,20 @@ config MODEM_GSM_MANUAL_MCCMNO
 	  the network provider and may need to be changed if auto is not
 	  selected.
 
+config MODEM_GSM_RSSI_WORK_PERIOD
```

Comment:
Better would be call this `MODEM_GSM_RSSI_POLLING_PERIOD` or similar, as that immediately tells reader what the variable is controlling.

Reply:
thanks for the review. addressed now